.github/workflows/test.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build and Publish Test
+on:
+  push:
+    branches:
+      - main
+      
+jobs:
+  terraform-plan-and-apply:
+    defaults:
+      run:
+        working-directory: ./config
+    runs-on: ubuntu-latest
+    env:
+      KEYCLOAK_CLIENT_ID: ${{ secrets.tf_test_kc_client_id }}
+      KEYCLOAK_CLIENT_SECRET: ${{ secrets.tf_test_kc_client_secret }}
+      KEYCLOAK_REALM: ${{ secrets.tf_test_kc_realm }}
+      KEYCLOAK_URL: ${{ secrets.tf_test_kc_url }}
+      TF_VAR_callisto_realm: ${{ secrets.tf_test_kc_realm }}
+      TF_VAR_callisto_url: ${{ secrets.tf_test_callisto_url }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: hashicorp/setup-terraform@v2
+
+    - name: Terraform Workspace
+      id: workspace
+      run: terraform workspace test
+
+    - name: Terraform Init
+      id: init
+      run: terraform init -reconfigure -backend-config="region=${{ secrets.tf_be_region }}" -backend-config="access_key=${{ secrets.tf_be_access_key }}" -backend-config="secret_key=${{ secrets.tf_be_secret_key }}" -backend-config="bucket=${{ secrets.tf_be_bucket }}" -backend-config="key=${{ secrets.tf_be_bucket_key }}"
+
+    - name: Terraform Validate
+      id: validate
+      run: terraform validate
+
+    - name: Terraform Plan
+      id: plan
+      run: terraform plan -input=false -out=tfplan
+
+    - name: Terraform Apply
+      id: apply
+      run: terraform apply -input=false -auto-approvetfplan
```

Comment:
Missing an equals here? `-auto-approve=tfplan`

Reply:
That's a missing space there, added it